core/Piwik.php

```diff
@@ -889,4 +889,14 @@ public static function translate($translationId, $args = array(), $language = nu
 
         return $translator->translate($translationId, $args, $language);
     }
+
+    public static function getPeriod($default = null)
+    {
+        return Common::getRequestVar('period', $default, 'string');
+    }
+
+    public static function getDate( $default = false)
```

Comment:
The default should be null here as well

```suggestion
    public static function getDate($default = null)
```